packages/js/src/components/AreaChart.js

```diff
@@ -1,39 +1,40 @@
+/* eslint-disable complexity */
 /* External dependencies */
-import PropTypes from "prop-types";
 import { Fragment } from "@wordpress/element";
+import PropTypes from "prop-types";
 
 /* Internal dependencies */
 import AreaChartTable from "./AreaChartTable";
 
 /**
  * Renders a SVG area chart.
  *
- * @param {array}    data                      Array of objects with X and Y coordinates for the SVG chart points.
- * @param {number}   width                     The SVG chart width.
- * @param {number}   height                    The SVG chart height.
- * @param {string}   fillColor                 The SVG chart area background color in HEX format.
- * @param {string}   strokeColor               The SVG chart line color in HEX format.
- * @param {number}   strokeWidth               The SVG chart line width.
- * @param {string}   className                 The CSS class name for the chart.
- * @param {Function} mapChartDataToTableData   Function to adapt the chart points to meaningful data for the table.
- * @param {string}   dataTableCaption          The data table caption text.
- * @param {array}    dataTableHeaderLabels     The text to be used for the data table headers.
- * @param {boolean}  isDataTableVisuallyHidden Whether the data table is viually hidden.
+ * @param {Array} data Array of objects with X and Y coordinates for the SVG chart points.
+ * @param {number} width The SVG chart width.
+ * @param {number} height The SVG chart height.
+ * @param {?string} [fillColor=null] The SVG chart area background color in HEX format.
+ * @param {string} [strokeColor="#000000"] The SVG chart line color in HEX format.
+ * @param {number} [strokeWidth=1] The SVG chart line width.
+ * @param {string} [className=""] The CSS class name for the chart.
+ * @param {Function} [mapChartDataToTableData=null] Function to adapt the chart points to meaningful data for the table.
```

Comment:
I suppose `{?Function}` is not a thing

Reply:
It is a thing, or `Function|null`, so need to fix 😄